src/wallet/rpcwallet.cpp

```diff
@@ -416,6 +416,12 @@ UniValue sendtoaddress(const JSONRPCRequest& request)
             "                             transaction, just kept in your wallet.\n"
             "5. subtractfeefromamount  (boolean, optional, default=false) The fee will be deducted from the amount being sent.\n"
             "                             The recipient will receive less bitcoins than you enter in the amount field.\n"
+            "6. replaceable            (boolean, optional) Allow this transaction to be replaced by a transaction with higher fees via BIP 125\n"
+            "7. conf_target            (numeric, optional) Confirmation target (in blocks)\n"
+            "8. \"estimate_mode\"      (string, optional, default=UNSET) The fee estimate mode, must be one of:\n"
```

Comment:
Hmm..  Yes, I think it might make sense to change this to `fee_mode`, but leave it `estimate_mode` for estimateSmartFee (done in another PR).  Even though they take the same values now, I could imagine they would differ in the future.  Any other thoughts?